resources/views/settings/ldap.blade.php

```diff
@@ -363,7 +363,7 @@
                                 {{ Form::label('ldap_emp_num', trans('admin/settings/general.ldap_emp_num')) }}
                             </div>
                             <div class="col-md-9">
-                                {{ Form::text('ldap_emp_num', Request::old('ldap_emp_num', $setting->ldap_emp_num), ['class' => 'form-control','placeholder' => 'employeenumber/employeeid', $setting->demoMode]) }}
+                                {{ Form::text('ldap_emp_num', Request::old('ldap_emp_num', $setting->ldap_emp_num), ['class' => 'form-control','placeholder' => trans('general.example') .'employeenumber/employeeid', $setting->demoMode]) }}
```

Comment:
Instead of the `/` can we say ` or ` in whatever language is appropriate?

Reply:
That's a little more complicated, since `blah or blah` doesn't work the same in every language. We'd need to add a specific translation string for `employeenumber or employeeid` and I'd worry that translators might try to literally translate the values for `employeenumber` and `employeeid`, which would make the problem worse. :-/